tests/test_user_mgt.py

```diff
@@ -102,12 +103,18 @@ def _check_user_record(user, expected_uid='testuser'):
 
 class TestAuthServiceInitialization:
 
+    def test_default_timeout(self, user_mgt_app):
+        auth_service = auth._get_auth_service(user_mgt_app)
+        user_manager = auth_service.user_manager
+        assert user_manager._client.timeout == _http_client.DEFAULT_TIMEOUT_SECONDS
```

Comment:
Hm... It seems like all of our components will have an approximately identical test like this one. We could remove them all and just add this into the _http_client tests. Or we could add a parameterized test that verifies that each component sets the timeout appropriately. (No action required; I'm just thinking out loud.)

Reply:
Yes, and I think the existing test cases in _http_client already provides us sufficient coverage. In each of the other modules it's probably enough to check that the Client.timeout is correctly set. We can do that clean up in a future iteration.